content/news/2024-07-01-GCC2024/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Galaxy Community Conference 2024 - a brief summary!
```

Comment:
Maybe highlight that this is a Freiburg summary?

Reply:
e.g.: "... - a brief summary from the Freiburg Galaxy Team"